includes/class-wc-stripe-blocks-support.php

```
@@ -111,48 +131,129 @@ public function get_payment_method_script_handles() {
-			array_merge( [ 'stripe' ], $dependencies ),
-			$version,
+			array_merge( [ 'stripe' ], $asset['dependencies'] ),
+			$asset['version'],
 			true
 		);
 		wp_set_script_translations(
-			'wc-stripe-blocks-integration',
+			self::BLOCKS_SCRIPT_HANDLE,
 			'woocommerce-gateway-stripe'
 		);
 	}
```

PHPMD (2.15.0)

[error] The method register_upe_payment_method_script_handles is not named in camelCase. (undefined)

(CamelCaseMethodName)